src/vivarium_nih_us_cvd/components/observers.py

```diff
@@ -5,22 +5,19 @@
 from vivarium_public_health.metrics.stratification import (
     ResultsStratifier as ResultsStratifier_,
 )
-from vivarium_public_health.utilities import EntityString, to_years
+from vivarium_public_health.utilities import EntityString, TargetString, to_years
 
 from vivarium_nih_us_cvd.constants import data_values
 
 
-class ResultsStratifier(ResultsStratifier_):
+class SimpleResultsStratifier(ResultsStratifier_):
```

Comment:
does "simple" effectively mean non-paf?

Reply:
I was going to call it the `PAFResultsStratifier`, but then I had to inherit from it for the "normal" results stratifier for the model due to the `get_age_bins` function being overridden. It's just the same results stratifier, but with the additional custom stratifications not added.